Resources/views/CRUD/base_list.html.twig

```diff
@@ -295,7 +295,13 @@ file that was distributed with this source code.
                         </div>
 
                         {% for paramKey, paramValue in admin.persistentParameters %}
-                            <input type="hidden" name="{{ paramKey }}" value="{{ paramValue }}">
+                            {% if paramValue is iterable %}
+                                {% for key, value in paramValue %}
+                                    <input type="hidden" name="{{ paramKey }}[{{ key }}]" value="{{ value }}">
+                                {% endfor %}
```

Comment:
No, I did not add this complexity.
Le 28 nov. 2015 20:41, "Christian Gripp" <notifications@github.com> a
écrit :

> In Resources/views/CRUD/base_list.html.twig
> <https://github.com/sonata-project/SonataAdminBundle/pull/3435#discussion_r46083571>
> :
>
> > @@ -295,7 +295,13 @@ file that was distributed with this source code.
> >                          </div>
> >
> >                          {% for paramKey, paramValue in admin.persistentParameters %}
> > -                            <input type="hidden" name="{{ paramKey }}" value="{{ paramValue }}">
> > +                            {% if paramValue is iterable %}
> > +                                {% for key, value in paramValue %}
> > +                                    <input type="hidden" name="{{ paramKey }}[{{ key }}]" value="{{ value }}">
> > +                                {% endfor %}
>
> Does this work if you have an array of arrays as a persistent parameter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonata-project/SonataAdminBundle/pull/3435/files#r46083571>
> .
>